release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1419,9 +1419,9 @@ This release introduces the ability to deploy to new {ibm-power-name} Virtual Se
 
 [discrete]
 [id="ocp-4-16-power-vs-capi_{context}"]
-=== {ibm-power-name} Virtual Verver cluster API updated to 0.8.0
+=== {ibm-power-name} Virtual Server updated to use {cap-ibm-first} 0.8.0
 
-With this release, the {ibm-power-name} VS CAPI has been updated to version 0.8.0.
+With this release, the {ibm-power-name} Virtual Server has been updated to use {cap-ibm-first} version 0.8.0.
```

Comment:
@hamzy does this updated wodring make sense wrt https://github.com/openshift/installer/pull/8458? Rendered text will be:

> **IBM Power® Virtual Server updated to use Cluster API Provider IBM Cloud 0.8.0**
> 
> With this release, the IBM Power® Virtual Server has been updated to use Cluster API Provider IBM Cloud version 0.8.0.

Reply:
Yes, this sounds reasonable.  Sorry for the delay. GH email is spammy.

Reply:
thanks!